net-dialup/freeradius-client/freeradius-client-1.1.6.ebuild

```diff
@@ -17,15 +18,20 @@ DEPEND="!net-dialup/radiusclient
 	!net-dialup/radiusclient-ng"
 RDEPEND="${DEPEND}"
 
+src_prepare() {
+	epatch "${FILESDIR}/${P}-BJA-cross.diff"
+	eautoreconf
+}
+
 src_configure() {
 	econf \
 		$(use_enable scp) \
 		$(use_enable shadow) \
-		--with-secure-path || die "econf failed"
+		--with-secure-path
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die "emake install failed"
+	emake install
```

Comment:
Done in last PR, that does match another comment in https://github.com/bjacquin/gentoo/commit/5b5090d46b1a8d482576ac0703cab3c2674a68a8#commitcomment-13699166